mvp.feature

```diff
@@ -661,11 +668,47 @@ Feature: MVP features for NVA
     And they select values for uploaded files
     And they are on the Submission tab
     When they click Publish
-    Then they are redirected to the Publication public page
+    Then they are redirected to the Public Page for Publication
     And they see the Publication confirmation message
 
+  @881
+  Scenario: View Public Page for Publication
+    Given the user has opened NVA
+    And see the Start Page or Public Profile
```

Comment:
Should be a scenario outline: and they are on <page>, Examples: Start page, Public Profile